app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -62,47 +63,43 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
 
   @override
   Future<Body> get() async {
-    const String projectId = 'flutter-dashboard';
-    const String dataset = 'cocoon';
-    const String table = 'Checklist';
-
-    final GitHub github = await config.createGitHubClient();
+    final GithubService githubService = await config.createGithubService();
     const RepositorySlug slug = RepositorySlug('flutter', 'flutter');
-    final Stream<RepositoryCommit> commits =
-        github.repositories.listCommits(slug);
+    const List<String> branches = <String>['master', 'beta', 'stable'];
```

Comment:
We're going to be using a release branch scheme, where each dev (`x.y.z`) release gets its own branch, and testing is done on those branches.  Then dev, beta, and stable will just point to hashes that have already been tested as part of the release branch.

So rather than enumerating the branches here, you'll want to query GitHub for the list of branches, then filter that list to include only branches that match a specified list of patterns (e.g. ['master', r'v[0-9]+\.[0-9]+\.[0-9]+']).

Reply:
I see there are existing branches: `vx.x.x-hotfixes` in flutter repo. Shall we exclude those, namely only strictly matching `vx.x.x` pattern?

Reply:
You can exclude them.  We'll want a way to update the pattern of the branches we match though, since the proposal for those branch names is still in flux.  Maybe a config file that we query so we can update the pattern without needing to update Cocoon?

Reply:
Should we add a top level folder called release to both flutter/flutter and flutter/engine with a file called branch_regs.txt with one regex per line?

Reply:
I'd put it somewhere in the `dev/` folder hierarchy.

Reply:
added.